config/confighttp/confighttp.go

```diff
@@ -351,6 +354,15 @@ func (hss *ServerConfig) ToServer(host component.Host, settings component.Teleme
 		o(serverOpts)
 	}
 
+	if hss.MemoryLimiter != nil {
+		ml, err := getMemoryLimiterExtension(hss.MemoryLimiter, host.GetExtensions())
+		if err != nil {
+			return nil, err
```

Comment:
Please cover this path to make codecov happy. Should be easy I guess

Reply:
covered